recompose-flow.md

```diff
@@ -1,19 +1,19 @@
-# Flow support in recompose
+# Flow support in Recompose
 
 ## History
 
-One month ago official flow library definition was landed in recompose.
-It was a long way before we added it having that [PR](https://github.com/acdlite/recompose/pull/241) was added by @GiulioCanti one year ago.
+One month ago the official Flow library definition was landed in Recompose.
+It was a long time coming, considering the original [PR](https://github.com/acdlite/recompose/pull/241) was created by @GiulioCanti a year ago.
```

Comment:
"@GiulioCanti" should either be a link (to his Twitter or Github or whatever) or just be his name formatted normally, "Giulio Canti".